platformio.ini

```diff
@@ -104,3 +104,19 @@ framework = arduino
 lib_deps = ${common.lib_deps}
 build_flags = ${common.build_flags}
 src_filter = ${common.src_filter} +<examples/sonoff-b1/sonoff-b1.cpp>
+
+[env:tx20sensor]
+platform = espressif8266
+board = nodemcuv2
+framework = arduino
+lib_deps = ${common.lib_deps}
+build_flags = ${common.build_flags}
+src_filter = ${common.src_filter} +<examples/tx20-sample/tx20-sample.cpp>
```

Comment:
Please only add one of the esp32/esp8266 versions. Each environment makes the test scripts compile all of esphomelib's code base another time.

Reply:
Fixed.